Resources/Prototypes/Floof/Entities/Mobs/Cyborgs/quadborg.yml

```diff
@@ -0,0 +1,38 @@
+
+- type: entity
+  id: BorgChassisQuad
+  parent: BaseBorgChassisNT
+  components:
+  - type: Sprite
+    sprite: Floof/Mobs/Silicon/quad.rsi
+    scale: 0.6, 0.6
+    layers:
+    - state: quadsprite
+      map: ["movement"]
+    - state: quadspritel
+      map: ["enum.BorgVisualLayers.Light"]
+      shader: unshaded
+      visible: false
+    - state: quadspritel
+      shader: unshaded
+      map: ["light"]
+      visible: false
+  - type: BorgChassis
+    maxModules: 6
+    moduleWhitelist:
+      tags:
+      - BorgModuleGeneric
+      - BorgModuleSecurity
+    hasMindState: quadsprite
+    noMindState: quadspritel
+  - type: AccessReader
+    access: [["Security"], ["Armory"], ["Command"]]
+  - type: BorgTransponder
+    sprite:
+      sprite: Floof/Mobs/Silicon/quad.rsi
+      state: quadsprite
+    name: cyborg
+  - type: Construction
+    node: cyborg
+  - type: Speech
+    speechVerb: Robotic
```

Comment:
```suggestion
```

Uneeded and this will cause issues with the current cyborg build, Speechverb is already on parent.

Reply:
I literally had to add this back because it threw errors.

Reply:
Huh, strange, il look into it.